odoo/openupgrade/doc/source/modules90-100.rst

```diff
@@ -72,7 +72,7 @@ Status :
 +-----------------------------------+-----------------------------------+
 |base_import_module                 |                                   |
 +-----------------------------------+-----------------------------------+
-|base_setup                         |                                   |
+|base_setup                         | Nothing to do                     |
```

Comment:
Well, it can be barely consider as nothing to do, as this is only have side-effects sometimes, but I prefer to stay safe with the previous known issues. Anyway, my comment was for isolating this kind of changes from PR with the scripts of one module. We can make a PR for these changes, or directly a commit including this documentation (as it doesn't affect Travis status and thus doesn't require even optimistic merging).